blivet/devices/lvm.py

```diff
@@ -356,6 +356,18 @@ def _remove_log_vol(self, lv):
     def _add_parent(self, parent):
         super(LVMVolumeGroupDevice, self)._add_parent(parent)
 
+        # we are creating new VG or adding a new PV to an existing one
+        if not self.exists or self._complete:
```

Comment:
This `or self._complete` looks quite arbitrary here.

Reply:
That's because `_add_parent` is used also during populate for existing VGs and we want to skip this check for existing "broken" VGs but we also want to run this check when adding a brand new PV to an existing (complete) VG. I've changed the if and the comment a little, I hope it makes more sense now.